### PR TITLE
cassandra plugin fix - loading config

### PR DESCRIPTION
**Pull request by mpundlik**
Added code sample in documentation for timeouts

### CODE REVIEW

File: db/sql/cassandra/README.md [hunk 1]
```diff
@@ -22,6 +22,18 @@ Also, the client can configure following timeouts:
 - RedialInterval
     - If not zero, gocql attempt to reconnect known DOWN nodes in every ReconnectSleep (ReconnectInterval)
     - Default value is 60s
+- Example
+```go
+    config := &cassandra.Config{
+        Endpoints:      endpoints,
```
Comment (jozef-slezak) [inline db/sql/cassandra/README.md]: please put to the example real string with IP & port

File: db/sql/cassandra/README.md [hunk 2]
```diff
@@ -22,6 +22,18 @@ Also, the client can configure following timeouts:
...
+        Port:           port,
```
Comment (jozef-slezak) [inline db/sql/cassandra/README.md]: We have agreed on that you will unify port to be consistent with etcd, redis...

Please refactor...

File: db/sql/cassandra/README.md [hunk 3]
```diff
@@ -22,6 +22,18 @@ Also, the client can configure following timeouts:
...
+        DialTimeout:    600000000,
+        OpTimeout:      60,
+        RedialInterval: 60000000000,
```
Comment (jozef-slezak) [inline db/sql/cassandra/README.md]: Please use multiply time... (it is not readable)

File: db/sql/cassandra/README.md [hunk 4]
```diff
@@ -22,6 +22,18 @@ Also, the client can configure following timeouts:
...
+    }
+
+    clientConfig, err2 := cassandra.ConfigToClientConfig(config)
```
Comment (jozef-slezak) [inline db/sql/cassandra/README.md]: err2 is strange...

File: db/sql/cassandra/README.md (hunk 1 above)
Comment (mpundlik) [reply db/sql/cassandra/README.md]: done.

File: db/sql/cassandra/README.md (hunk 2 above)
Comment (mpundlik) [reply db/sql/cassandra/README.md]: Will refactor in a separate commit

File: db/sql/cassandra/README.md (hunk 3 above)
Comment (mpundlik) [reply db/sql/cassandra/README.md]: done

File: db/sql/cassandra/README.md (hunk 4 above)
Comment (mpundlik) [reply db/sql/cassandra/README.md]: done

**Pull request merged**